src/components/Navbar.js

```diff
@@ -33,54 +33,48 @@ export class Navbar extends Component {
           >
             <span className="navbar-toggler-icon" />
           </button>
-          <div className="collapse navbar-collapse " id="navbarA">
-            {this.props.loggedIn && (
-              <ul className="navbar-nav ml-auto" id="navList">
-                <li className="nav-item nav-link" >
-                  <a
-                    className="nav-link text-white"
-                    href="/home"
-                  >
-                    <strong>
+          <div className="container">
+            <div className="collapse navbar-collapse row" id="navbarA">
+              {this.props.loggedIn && (
+                <ul className="navbar-nav ml-auto col-md-4" id="navList">
+                  <li className="nav-item nav-link col-md-2">
```

Comment:
Don't give these links bootstrap column classes because they will take on `col-md-12` the moment the screen width reduces `col-md` instead of letting the button take over.

Reply:
I have removed them